### PR TITLE
Add Contact to FD API

### PR DESCRIPTION
**Pull request by gsnovak**
We should be able to pull all the information we need off of this.

### CODE REVIEW

**Review by fiestacasey — DISMISSED** [review]
Bump version
https://github.com/wantable/freshdesk-ruby/blob/master/lib/freshdesk/version.rb

and then bundle install

add yourself to authors:
https://github.com/wantable/freshdesk-ruby/blob/master/freshdesk.gemspec#L10

tests would be nice but i'm okay without them and just testing after this is updated with our repo

File: lib/freshdesk/contact.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+      def list_all_contacts(query: "")
+        new("/contacts#{query}").resource.get
+      end
+
+      def update_a_contact(id:, params: {})
+        new("/contacts/#{id}").resource(id: id, params: params).put
+      end
+
+      def delete_a_contact(id:)
```
Comment (ksolkowski) [inline lib/freshdesk/contact.rb:22]: the : on the other side is weirding me out lol

**Review by austinkahly — APPROVED** [review]

File: lib/freshdesk/contact.rb (hunk 1 above)
Comment (gsnovak) [reply lib/freshdesk/contact.rb:22]: id:

**Comment by austinkahly** [comment]
@gsnovak Do you have to require this in `lib/freshdesk.rb`?

**Pull request merged**